examples/files.json

```diff
@@ -141,6 +141,7 @@
 		"webgl_materials_displacementmap",
 		"webgl_materials_envmaps",
 		"webgl_materials_envmaps_exr",
+		"webgl_materials_envmaps_ground-projected",
```

Comment:
I suggest to replace `-` in the example name with `_`.

Reply:
I believe that would make the sidebar section title `materials / envmaps / ground / projected` and not `materials / envmaps / ground-projected`